typesutil/types.go

```diff
@@ -0,0 +1,57 @@
+package typesutil
```

Comment:
I'm something implementing this functions? Or it is being used for some type conversions?

Reply:
Nevermind I see know - it is implicitly called for type conversion in the struct connection 👍